test_s3.py

```diff
@@ -1,29 +1,45 @@
 import boto3
```

Comment:
add main method and parse config file name as argument. use [argparse](https://docs.python.org/3/library/argparse.html) module to parse argument.
Config file should be in yaml format with level and type of logging. use `pyyaml` module for yaml parsing. 
Add `.gitignore` file, `requirements.txt` with all external dependencies and `README.md`.

Reply:
done